Project-Aurora/Project-Aurora/Profiles/Stardew Valley/GSI/Nodes/GameStateNode.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Aurora.Profiles.StardewValley.GSI.Nodes {
+    public enum GameStates
+    {
+        Unknown = -1,
+        TitleMenu,
+        Loading,
+        InGame
+    }
+
+    public enum GameModes
+    {
+        Unknown = -1,
+        TitleScreen,
+        LoadScreen,
+        NewGame,
+        PlayingGame,
+        LogoScreen,
+        Loading = 6,
+        Save,
+        SaveComplete,
+        SelectGame,
+        Credits,
+        ErrorLog
+    }
+
+    public class GameStateNode : AutoJsonNode<GameStateNode>
+    {
+        public GameStates GameState;
+        public string CurrMenu;
```

Comment:
Is Curr short for current? 
If so, i would just say Current since this is something the user will see. There's no real need to save on characters imo.